.travis.yml

```diff
@@ -1,23 +1,16 @@
 language: julia
 os:
   - linux
+dist: xenial
```

Comment:
Why add this? Personally I think it's fine to let Travis use whatever its default Linux distribution is (which is currently Xenial).

Reply:
For sure! I saw it being used in other packages in the beacosystem and thought there may be a consistency reason for it (I'm basing most CI choices here off OndaEDF, which may not be the wave). I'll revert if need be!

Reply:
Reverted here: https://github.com/beacon-biosignals/EDF.jl/pull/26/commits/d51965b6ea43729cbb69d00e32ab9bc04dd8ded7